pypesto/hierarchical/problem.py

```diff
@@ -164,6 +164,48 @@ def check_edatas(self, edatas: List[amici.ExpData]) -> bool:
 
         return True
 
+    def check_simulation_edatas(
```

Comment:
What's the intention here? Avoiding that one accidentally passes different datasets?

There would be some members to be checked, see https://amici.readthedocs.io/en/latest/generated/amici.amici.html#amici.amici.ExpData

Reply:
The idea is to check that the only difference between simulation_edatas and the original edatas is that simulation edatas might consist of more timepoints than the original ones in order to obtain a more dense simulation. I am not sure whether it is necessary to have this check since I would not expect someone to provide simulation edatas by themselves.